Gemfile.lock

```diff
@@ -458,8 +458,8 @@ GEM
     rubyzip (2.3.2)
     rufus-scheduler (3.6.0)
       fugit (~> 1.1, >= 1.1.6)
-    sdoc (2.2.0)
-      rdoc (>= 5.0)
+    sdoc (2.3.0)
```

Comment:
Can we update the gemfile to use `>= 2.3.0`?

Reply:
Sure, will do.